daemons/gptp/common/ptp_message.cpp

```diff
@@ -465,10 +464,27 @@ PTPMessageCommon *buildPTPMessage
 			msg = annc;
 		}
 		break;
+
+	case SIGNALLING_MESSAGE:
+		{
+			PTPMessageSignalling *signallingMsg = new PTPMessageSignalling();
```

Comment:
sometimes we check for malloc (new) returns, sometimes we dont. Is there a best practice item for this within open-avb?